packages/yourloops/lib/auth/hook/use-auth.tsx

```diff
@@ -0,0 +1,96 @@
+import { createContext, useContext, useState } from "react";

```

Comment:
```suggestion
/**
 * Copyright (c) 2021, Diabeloop
 * Hook for auth API
 *
 * This program is free software; you can redistribute it and/or modify it under
 * the terms of the associated License, which is identical to the BSD 2-Clause
 * License as published by the Open Source Initiative at opensource.org.
 *
 * This program is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 * FOR A PARTICULAR PURPOSE. See the License for more details.
 *
 * You should have received a copy of the License along with this program; if
 * not, you can obtain one from Tidepool Project at tidepool.org.
 */
import * as React from "react";
```